addon/helpers/format-money.ts

```diff
@@ -22,7 +22,8 @@ var _formatMoney = function (
 };
 
 export function formatMoneyHelper(params: any[]) {
-  const [amount, currency, format = 'raw', compact = false] = params;
+  let [amount, currency, format = 'raw', compact = false] = params;
+  if ((amount < 1000 && format === 'raw') || (amount < 100000 && format === 'cents')) compact = false;
```

Comment:
what about having a `ANTI_COMPACT_RAW_THRESHOLD = 1000` & `ANTI_COMPACT_CENTS_THRESHOLD = 100_000` consts to make this clearer?

Reply:
`THRESHOLD` -> `CEIL` ?